### PR TITLE
[double-take] feat(double-take): initial commit

### PR DESCRIPTION
**Pull request by crutonjohn**
double-take is a unified UI and API for processing and training images for facial recognition. it
can be used in conjunction with frigate and home-assistant.

https://github.com/jakowenko/double-take

**Description of the change**

Addition of [double-take](https://github.com/jakowenko/double-take)

**Benefits**

Being able to leverage double take in k8s to enhance frigate and home-assistant

**Possible drawbacks**

facial detection never works :(

**Applicable issues**

- fixes # **N/A**

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/double-take/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# double-take
+
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![AppVersion: 1.0.0](https://img.shields.io/badge/AppVersion-1.0.0-informational?style=flat-square)
```
Comment (rwaltr) [inline charts/stable/double-take/README.md:3]: Appversion appears to really be 1.16.0

File: charts/stable/double-take/README.md.gotmpl [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+{{- define "custom.helm.url" -}}
+https://k8s-at-home.com/charts/
+{{- end -}}
+
+{{- define "custom.helm.path" -}}
+{{ template "custom.repository.organization" . }}/{{ template "chart.name" . }}
+{{- end -}}
+
+{{- define "custom.notes" -}}
+**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/k8s-at-home/charts/issues/new/choose)**
```
Comment (rwaltr) [inline charts/stable/double-take/README.md.gotmpl:18]: Lines 38-42 of Values.yaml Could be added here as well

File: charts/stable/double-take/README.md (hunk 1 above)
Comment (rwaltr) [reply charts/stable/double-take/README.md:3]: Dont forget to regen readme :D

**Review by rwaltr — COMMENTED** [review]
Only a few small suggestions

File: charts/stable/double-take/README.md (hunk 1 above)
Comment (crutonjohn) [reply charts/stable/double-take/README.md]: it's actually 1.6.0 from what i can tell; updated!

File: charts/stable/double-take/README.md.gotmpl (hunk 2 above)
Comment (crutonjohn) [reply charts/stable/double-take/README.md.gotmpl:20]: added!

**Comment by onedr0p** [comment]
Looks good thanks @crutonjohn and thanks for the review @rwaltr

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
Not sure why pre-commit is failing, maybe an extra new line?

**Comment by crutonjohn** [comment]
> Not sure why pre-commit is failing, maybe an extra new line?
> 
> Edit: Looks like the generated README is weird, did you generate it with our helm-docs wrapper?

`task chart:helm-docs CHART_TYPE=stable CHART=double-take` overwrites the `README.md.gotmpl`, so I'm not really sure how to proceed or what task i need to run so that it updates the README without replacing the template?

**Comment by onedr0p** [comment]
> Afterwards you can run ./hack/gen-helm-docs.sh stable <chart> again to update the chart's README.md file.

**Comment by crutonjohn** [comment]
> > Afterwards you can run ./hack/gen-helm-docs.sh stable  again to update the chart's README.md file.

```
❯ ./hack/gen-helm-docs.sh stable double-take
-] Copying templates to /Users/crutonjohn/Projects/Personal/github/external/charts/charts/stable/double-take
INFO[2021-11-12T13:47:40-05:00] Found Chart directories [.]
INFO[2021-11-12T13:47:40-05:00] Generating README Documentation for chart /Users/crutonjohn/Projects/Personal/github/external/charts/charts/stable/double-take
```

when it copies templates it overwrites the string that i added as per @rwaltr's suggestion [here](https://github.com/k8s-at-home/charts/pull/1271#discussion_r747707129)

**Comment by onedr0p** [comment]
Anything custom you want in the README should be added to the `README_CONFIG.md.gotmpl` template file. The `README.md.gotmpl` is the same for all applications.

**Pull request merged**